README.md

```diff
@@ -164,3 +179,28 @@ by Fika.
 If you'd like to be part of the Fika community, please consider joining
 our [Discord server](https://discord.gg/zNs6Gs5). That's the best place to
 ask questions or get guidance on contributing to Fika.
+
+## License
+
+Licensed under the [MIT License](LICENSE)
+
+## Contributors
+
+| <a href="http://emilsoman.com/"><img src="https://avatars.githubusercontent.com/emilsoman" width="150px" height="150px" /></a> |
+| :----------------------------------------------------------------------------------------------------------------------------: |
+|                                            **[Emil Soman](http://emilsoman.com/)**                                             |
+
+<hr/>
+
+<p align="center">
```

Comment:
Should we link to Contributors.md instead?
No point in redundant doc 🙂

Reply:
I have added that just to show the main author of the repo , rest of contributors will be added in Contributors.md.

Reply:
Then let's change the heading to Author?

Reply:
Yeah right